### ISSUE TITLE
How we can select translation?

### ISSUE THREAD

**Issue by anikitin35** [nathanp/crypto-price-widget#15]
Missing UI function.

**Comment by nathanp** [comment]
Hi @anikitin35, thanks for the suggestion! Currently, this app is English only. It'd be great to add more languages down the line. If you would like to contribute please feel free to do so :)

**Comment by anikitin35** [comment]
Yes, I would like to contribute Russian version. Best regards! Please tell me an instructions and partnership information. Telegram +7-911-5013-94-44

**Comment by wertzuiop123** [comment]
In which files translations are designated?

are there specific code fragments for it?
e.g. <translation><translation> ?

could add german for you
